ccloud-observability/start.sh

```diff
@@ -79,6 +77,28 @@ echo "METRICS_API_SECRET=$METRICS_API_SECRET" >> .env
 echo "CLOUD_CLUSTER=$CLOUD_CLUSTER" >> .env
 echo "BOOTSTRAP_SERVERS=$BOOTSTRAP_SERVERS" >> .env
 echo "SASL_JAAS_CONFIG=$SASL_JAAS_CONFIG" >> .env
+echo "CLOUD_CONNECTORS=$CLOUD_CONNECTORS" >> .env
+echo "CLOUD_KSQLDB_APPS=$CLOUD_KSQLDB_APPS" >> .env
+echo "CLOUD_SCHEMA_REGISTRY=$CLOUD_SCHEMA_REGISTRY" >> .env
+
+# Prometheus does not support env/variables in config yaml, https://github.com/prometheus/prometheus/issues/2357
```

Comment:
Maybe add a comment here that explains the purpose of this code, which IIUC is to create custom `monitoring_configs/prometheus/prometheus.yml`